### ISSUE TITLE
Change db to oracle db it not not work

### ISSUE THREAD

**Issue by Hunglmc** [blazorhero/CleanArchitecture#460]
when i change  db to oracle with code internal static IServiceCollection AddDatabase(
            this IServiceCollection services,
            IConfiguration configuration)
            => services
                .AddDbContext<VRBSecureSalaryContext>(options => options
                    .UseOracle(configuration.GetConnectionString("OracleConnection")))
            .AddTransient<IDatabaseSeeder, DatabaseSeeder>();
 and change connect string to  oracle the system   **_Oracle.ManagedDataAccess.Client.OracleException: 'ORA-01435: user does not exist_** when i run migrate()

I tried using toad and connecting with normal connection information and didn't encounter any problems. Please let me know what problem I have or how to change to oracle db

**Comment by MomboMan** [comment]
What user name is in your connection string?  It should be the same as the one you are using when connecting with Toad.

**Comment by Hunglmc** [comment]
> What user name is in your connection string? It should be the same as the one you are using when connecting with Toad.
Yes, they are all the same and similar to toad. Perhaps it's because we need to add a user with a schema identity
